packages/circuit-ui/hooks/useMedia/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { useMedia } from './useMedia';
```

Comment:
Do we want to export this hook from Circuit?

Reply:
Maybe? Didn't wanna spend time on this now.

Reply:
All good, we can think about this in a future minor